packages/notebook/src/toc.ts

```diff
@@ -610,7 +610,8 @@ export class NotebookToCFactory extends TableOfContentsFactory<NotebookPanel> {
             console.debug('scrolling to heading: using fallback strategy');
             await widget.content.scrollToItem(
               widget.content.activeCellIndex,
-              this.scrollToTop ? 'start' : undefined
+              this.scrollToTop ? 'start' : undefined,
+              0
```

Comment:
Hmm, I never asked about this change (margin=0) and I'm not sure I understand why it's here

Reply:
Good question. First, this is the fallback strategy which is unlikely to be used. Before this PR it was broken in defer mode because of wrong node being used for scrolling further down the line. Currently both defer and full mode will work, invoking a logic which considers the margin. Now, the default visibility margin for scrolling on action is 25%. But when user uses ToC they often want to find the heading and if it was partially revealed and the clicked either way it means that they expect actions anyways.